src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,39 @@
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__elem:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  position: relative;
+  text-decoration: none;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+}
+
+.nav__link:hover {
+  color: $blue-hover;
+}
+
+.nav__link--is-active {
```

Comment:
The class 'nav__link--is-active' uses a non-standard naming convention for indicating an active state. Consider using a more consistent naming pattern like 'nav__link--active' to align with common BEM practices.